### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SerhiiUnhurian**
- [DEMO LINK](https://SerhiiUnhurian.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://SerhiiUnhurian.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,129 @@
...
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-family: "Roboto", sans-serif;
```
Comment (VitaliyBodnar) [inline src/style.css:30]: font-family is already applied to tag html, so it's redundant here

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,129 @@
...
+  --main-color: #060b35;
+  --secondary-color: #616070;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
```
Comment (VitaliyBodnar) [inline src/style.css:13]: box-sizing: border-box is duplicated, please move it to the `*{ box-sizing: border-box; }`

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,129 @@
...
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
```
Comment (VitaliyBodnar) [inline src/style.css:18]: '#fff' is duplicated, so make sense to move it to the variable

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,129 @@
...
+}
+
+.card__price-price {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__buyButton {
```
Comment (VitaliyBodnar) [inline src/style.css:109]: you need to use lower case for the class name

Comment (VitaliyBodnar) [inline src/style.css:109]:
```suggestion
.card__buy-button {
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,129 @@
...
+  --button-color: #00acdc;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+}
+
 body {
   margin: 0;
 }
+
+.card {
```
Comment (VitaliyBodnar) [inline src/style.css:12]: card is BEM block, so please move card and other block to the separate files

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/styles/blocks/reset.css [hunk 6]
```diff
@@ -0,0 +1,15 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
```
Comment (denys-danyliuk) [inline src/styles/blocks/reset.css:6]: 2'nd point in checklist

File: src/index.html [hunk 7]
```diff
@@ -5,9 +5,47 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </section>
+        <div class="card__reviews-count">
+          Reviews: 5
+        </div>
+      </div>
```
Comment (denys-danyliuk) [inline src/index.html:40]:
```suggestion
      <div class="card__reviews">
        <section class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </section>
        
        <div class="card__reviews-count">
          Reviews: 5
        </div>
      </div>
```

File: src/index.html [hunk 8]
```diff
@@ -5,9 +5,47 @@
...
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__reviews">
+        <section class="stars stars--4">
```
Comment (denys-danyliuk) [inline src/index.html:30]:
```suggestion
        <div class="stars stars--4">
```
here div would be fine

File: src/index.html [hunk 9]
```diff
@@ -5,9 +5,47 @@
...
+        class="card__img"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
```
Comment (denys-danyliuk) [inline src/index.html:27]: use p

File: src/index.html [hunk 10]
```diff
@@ -5,9 +5,47 @@
...
+        <section class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </section>
+        <div class="card__reviews-count">
+          Reviews: 5
+        </div>
```
Comment (denys-danyliuk) [inline src/index.html:39]: use p, 12'th point in checklist

File: src/styles/blocks/card.css [hunk 11]
```diff
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  height: 408px;
```
Comment (denys-danyliuk) [inline src/styles/blocks/card.css:3]: 8'th point in checklist

File: src/styles/blocks/card.css [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+.card__code {
+  color: var(--secondary-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
```
Comment (denys-danyliuk) [inline src/styles/blocks/card.css:33]: use sass nesting

File: src/index.html [hunk 13]
```diff
@@ -5,9 +5,47 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="iMac img"
```
Comment (denys-danyliuk) [inline src/index.html:17]: 13'th point in checklist

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/styles/blocks/card.css (hunk 12 above)
Comment (SerhiiUnhurian) [reply src/styles/blocks/card.css:33]: I coded this in css not in scss so i cant use nesting here i guess

File: src/index.html (hunk 13 above)
Comment (SerhiiUnhurian) [reply src/index.html]: okay

File: src/styles/blocks/card.css [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+
+.card__price-price {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__buy-button {
+  box-sizing: border-box;
```
Comment (mykhalenych) [inline src/styles/blocks/card.css:69]: you already added box-sizing in `*` selector, delete here

File: src/styles/blocks/card.css [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+  margin-bottom: 24px;
+  align-items: flex-end;
+}
+
+.card__reviews-count {
+  font-size: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  font-weight: 400;
```
Comment (mykhalenych) [inline src/styles/blocks/card.css:44]: also delete all font-weight: 400 because it is by default

**Review by mykhalenych — APPROVED** [review]

File: src/styles/blocks/card.css (hunk 14 above)
Comment (SerhiiUnhurian) [reply src/styles/blocks/card.css:69]: ++

**Pull request closed without merging**